taichi/backends/opengl/struct_opengl.cpp

```diff
@@ -94,7 +125,7 @@ size_t OpenglStructCompiler::compute_snode_size(const SNode &snode) {
     ch_size += compute_snode_size(*ch);
   }
   int n = snode.num_cells_per_container;
-  return n * ch_size + opengl_get_snode_meta_size(snode);
+  return alignment_bytes + n * ch_size + opengl_get_snode_meta_size(snode);
```

Comment:
Can we guarantee that the return value is also aligned with "element_stride"?

Reply:
I simplified the code a bit - now both mem_offset_in_root for the snode and return value here are guaranteed to be multiple of its element_stride.